### PR TITLE
Updated and converted i2samp script

### PR DESCRIPTION
**Pull request by makermelissa**
I updated the i2samp script to use the max98357a device tree overlay. While updating, I converted the script to python.

cc: @ladyada for review

### CODE REVIEW

**Comment by makermelissa** [comment]
@ladyada could I get a review? I'm holding off on finishing updating the guide until this is merged.

**Review by ladyada — APPROVED** [review]

**Comment by ladyada** [comment]
Sorry this got lost in my inbox.

**Pull request merged**
